docs/ref/django-admin.txt

```diff
@@ -870,11 +870,13 @@ are reserved for the superuser (root).
 This server uses the WSGI application object specified by the
 :setting:`WSGI_APPLICATION` setting.
 
-DO NOT USE THIS SERVER IN A PRODUCTION SETTING. It has not gone through
-security audits or performance tests. (And that's how it's gonna stay. We're in
-the business of making web frameworks, not web servers, so improving this
-server to be able to handle a production environment is outside the scope of
-Django.)
+.. warning:: DO NOT USE THIS SERVER IN A PRODUCTION SETTING.
+
+    This server has not gone through security audits or performance tests, and
+    hence is unsuitable for production. We're in the business of making web
+    frameworks, not web servers, so improving this server to be able to handle a
+    production environment is outside the scope of Django.
+
```

Comment:
```suggestion
    This lightweight development server has not gone through security audits or
    performance tests, hence is unsuitable for production. Making this server
    able to handle a production environment is outside the scope of Django.
```
I don't love the original "we're in the business of" sentence. I quite like how above it is described as "lightweight development" and so am adding this in a few places to emphasize it's purpose